ob_intern/Custom View/Progress Bar/UploadVideoProgressbar.swift

```diff
@@ -28,16 +57,25 @@ class UploadVideoProgressbar: UIView {
     
     private func setUpValues() {
         setupContainerView()
-        setupLabel()
+        setupLabelandImage()
```

Comment:
```suggestion
        setupLabelAndImage()
```
ฝากทำ Camel case จะได้อ่านง่ายขึ้นนิดดดดนึงครับ